### ISSUE TITLE
Proposed Check:  async function without await statements should be sync function

### ISSUE THREAD

**Issue by linw1995** [PyCQA/flake8-bugbear#159]
Async function like below `receive` has no await statements. It makes `foo` running forever without giving back control to the event loop.

```python
import asyncio

loop = asyncio.get_event_loop()

async def foo():
    async def receive():  # should warning or info, this can be a sync function
        return "haha"

    while True:
        data = await receive()


async def bar():
    while True:
        await asyncio.sleep(1)
        print("Hello")


loop.create_task(foo())
loop.create_task(bar())
loop.run_forever()
```

**Comment by cooperlees** [comment]
This feels to me it should possibly be in a dedicated flake8 asyncio related linting plugin. My quick google does not show one existing surprisingly, so I guess that's why you came here.

I'd like more opinions here. I'm open to it possibly, but I feel these would have to be opt in.

**Comment by cooperlees** [comment]
Moved to new `flake8-async` proposed plugin:
https://github.com/cooperlees/flake8-async/issues/2
